### PR TITLE
Editorial changes

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

**Review by aj-stein-nist — APPROVED** [review]

**Review by JAG-UK — APPROVED** [review]

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by roywill — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -485,8 +485,8 @@ Multiple Issuers can make the same Statement about a single Artifact, affirming
 At least one identifier for an identity document MUST be included in the protected header of the COSE Envelope, as one of `x5t` or `kid`.
 Additionally, `x5chain` that corresponds to either `x5t` or `kid` identifying the leaf certificate in the included certification path MAY be included in the unprotected header of the COSE Envelope.
 
-- When using x509, Support for `x5t` is mandatory to implement.
-- Support for `kid` in the protected header and `x5chain` in the unprotected heaer is optional.
+- When using x.509 certificates, support for `x5t` is mandatory to implement.
+- Support for `kid` in the protected header and `x5chain` in the unprotected header is optional to implement.
```
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:489]:
```suggestion
- Support for `kid` in the protected header and `x5chain` in the unprotected header is OPTIONAL to implement.
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -485,8 +485,8 @@ Multiple Issuers can make the same Statement about a single Artifact, affirming
 At least one identifier for an identity document MUST be included in the protected header of the COSE Envelope, as one of `x5t` or `kid`.
 Additionally, `x5chain` that corresponds to either `x5t` or `kid` identifying the leaf certificate in the included certification path MAY be included in the unprotected header of the COSE Envelope.
 
-- When using x509, Support for `x5t` is mandatory to implement.
-- Support for `kid` in the protected header and `x5chain` in the unprotected heaer is optional.
+- When using x.509 certificates, support for `x5t` is mandatory to implement.
```
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:488]:
```suggestion
- When using x.509 certificates, support for `x5t` is REQUIRED to implement.
```

**Review by henkbirkholz — APPROVED** [review]
fiddled with the language that intended to be normative and made it normative language

**Pull request merged**